packages/botonic-nlp/src/tasks/ner/process/processor.ts

```diff
@@ -0,0 +1,97 @@
+import { tensor, Tensor2D } from '@tensorflow/tfjs-node'
+
+import { Sample } from '../../../dataset/types'
+import { Codifier } from '../../../preprocess/codifier'
+import { PADDING_TOKEN, UNKNOWN_TOKEN } from '../../../preprocess/constants'
+import { Preprocessor } from '../../../preprocess/preprocessor'
+import { NEUTRAL_ENTITY } from './constants'
+import { InputData, OutputData } from './types'
+
+export class Processor {
+  constructor(
+    public preprocessor: Preprocessor,
+    readonly tokenCodifier: Codifier,
+    readonly entityCodifier: Codifier
+  ) {}
+
+  // Processes samples and generates the Input and Output data.
```

Comment:
now the comment says the same as the function prototype. So you can remove the comment :+1: